spec_util/testdata/meld/meld_nullable_1.pb.txt

```diff
@@ -0,0 +1,92 @@
+# api_spec.Witness proto
+
+method {
+  id {
+    api_type: HTTP_REST
+  }
+  responses {
+    key: "FiNRlcJh7SI="
+    value {
+      struct {
+        fields {
+          key: "left_present_right_present"
+          value {
+            primitive {
+              int64_value {
+              }
+            }
+          }
+        }
+        fields {
+          key: "left_present_right_none"
```

Comment:
```suggestion
          key: "left_present_right_null"
```
Similarly for other places where "none" really means "present but null".

Reply:
I'd prefer not to change this, because "none" matches the syntax of the IR's None object.  Instead, I added more test fields with `not_present` (e.g. `left_present_right_not_present`) to indicate the field is not present, rather than present but null.